tarodb/assets_store_test.go

```diff
@@ -248,6 +248,32 @@ func assetWitnessEqual(t *testing.T, a, b []asset.Witness) {
 	}
 }
 
+func assertAssetEqual(t *testing.T, a, b *asset.Asset) {
+	// Before comparison, we unset the split commitments, so we can compare
+	// them directly.
+	assetWitnessEqual(t, a.PrevWitnesses, b.PrevWitnesses)
```

Comment:
Could we use `a.DeepEqual(b)` here instead?

Reply:
Yep looks like it, I remember some other instance where it didn't work, but better to just use it here instead.